azure/template.json

```diff
@@ -327,6 +321,7 @@
     "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]",
     "appServicePlanName": "[concat(variables('resourceNamePrefix'), '-asp')]",
     "appServiceRuntimeStack": "[concat('DOCKER|', parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]",
+    "backgroundContainerImageReference": "[concat(parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]",
```

Comment:
Whether we chose `sidekiq` or `bg jobs`, perhaps we should align the name here with parameter name on line 120.

Reply:
done....